polyaxon/templates/api-deployment.yaml

```diff
@@ -21,6 +21,10 @@ metadata:
     app.kubernetes.io/managed-by: "helm"
 spec:
   replicas: {{ .Values.api.replicas }}
+  selector:
```

Comment:
Is the `selector` field required in the 1.16?

Reply:
Yes, it's required by apps/v1 spec.

Reply:
ok thanks for the fix then.